packages/contracts-ops/contracts/CallBatch.sol

```diff
@@ -16,32 +16,91 @@ abstract contract CallBatch is Script {
     using JsonWriter for JsonWriter.Buffer;
     using JsonWriter for string;
 
-    GovernanceMessage.Call[] public calls;
+    string localDomainName;
```

Comment:
bit of a hack. the rub here is that 
1. we have been using strings for domains within forge scripts thus far
2. the `GovernanceRouter`, like other contracts in the solidity codebase, use uint32 for domains 
3. `CallBatch` doesn't have access to `Config`, and can't convert from the forge scripts' strings to the contracts' expected uint32

Reply:
we should probably reconcile this problem. a potential solution is overloading all of the config getters to work with uint32 or string domains. we can write a config function which converts from a uint32 domain to a string by looping over each of the networks and to find the matching string for a domain.

Reply:
> we have been using strings for domains within forge scripts thus far

this is because the config lists networks by string, and has them keyed by string. i.e. our ops must have a string -> number mapping available to translate from config to solidity state

Reply:
>  a potential solution is overloading all of the config getters to work with uint32 or string domains.

sdk resolvedomain function